deis/builder
# fix(pkg/gitreceive/config.go): require dependent image names

# Summary of Changes

This PR requires the slug builder image and docker builder image environment variables, so that the builder will not start using old images.

# Issue(s) that this PR Closes

Closes https://github.com/deis/builder/pull/376
Closes https://github.com/deis/builder/issues/357

# Associated [End To End Test](https://github.com/deis/workflow-e2e) PR(s)

None

# Associated [Documentation](https://github.com/deis/workflow) PR(s)

None

# Associated [Design Document](http://docs.deis.io/en/latest/contributing/design-documents)(s)

None

# Testing Instructions

1. Create a Deis Cluster with a builder created with this patch.
2. Register an app
3. `git push` some code (https://github.com/deis/example-go for example) to your deis cluster

The `git push` should succeed and launch a slug builder or docker builder pod (depending on the type of repo that you pushed) specified in the helm chart.

# Pull Request Hygiene TODOs
